### PR TITLE
[ annotator ] 泛型...函数定义...

### PR DESCRIPTION
**Pull request by HoshinoTented**
QAQ

### CODE REVIEW

**Comment by ice1000** [comment]
CI build failed please fix

File: src/org/ice1000/devkt/ui/annotator.kt [hunk 1]
```diff
@@ -26,6 +26,18 @@ class KotlinAnnotator {
...
+			is KtTypeProjection -> {
+				//<泛型>
+				if (element.prevSibling?.node?.elementType == KtTokens.LT
+						&& element.nextSibling?.node?.elementType == KtTokens.GT) {
+					document.highlight(element, colorScheme.numbers)
+				}
+			}
+
+			is KtFunction -> {
+				document.highlight(element.nameIdentifier ?: return, colorScheme.functionDeclaration)
```
Comment (ice1000) [inline src/org/ice1000/devkt/ui/annotator.kt]: Are you sure `functionDeclaration` is defined? Do you have uncommited files?

**Comment by ice1000** [comment]
Superseded by #5

File: src/org/ice1000/devkt/ui/annotator.kt (hunk 1 above)
Comment (HoshinoTented) [reply src/org/ice1000/devkt/ui/annotator.kt]: No, I am not sure QAQ

**Pull request closed without merging**